recipes/behst/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.6" %}
+
+package:
+  name: behst
+  version: {{ version }}
+
+source:
+  fn: behst-{{ version }}.tar.gz
+  url: https://bitbucket.org/hoffmanlab/behst/get/tip.tar.bz2
```

Comment:
@davidechicco note the url is set to `tip` so upstream changes will get picked up here.